crates/types/src/cluster_controller.rs

```diff
@@ -33,10 +40,13 @@ pub enum ReplicationStrategy {
 #[serde_as]
 #[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
 pub struct SchedulingPlan {
-    version: Version,
+    pub version: Version,
     // flexbuffers only supports string-keyed maps :-( --> so we store it as vector of kv pairs
     #[serde_as(as = "serde_with::Seq<(_, _)>")]
-    partitions: BTreeMap<PartitionId, TargetPartitionState>,
+    pub partitions: BTreeMap<PartitionId, TargetPartitionState>,
+    // /// Replicated loglet target configuration.
+    #[serde_as(as = "serde_with::Seq<(_, _)>")]
+    pub logs: BTreeMap<LogId, TargetLogletState>,
```

Comment:
Is it important that the scheduling decision of PPs and logs are stored together? Would it be enough if the target state of the logs is stored in `Logs`?